jpa-hibernate-students-courses/src/main/java/dao/StudentDAO.java

```diff
@@ -0,0 +1,45 @@
+package dao;
+
+import tables.Student;
+import jakarta.persistence.EntityManager;
+import lombok.AllArgsConstructor;
+
+
+import java.util.List;
+
+@AllArgsConstructor
+public class StudentDAO implements DAOBasicCurd<Student> {
+    private EntityManager em;
```

Comment:
i know they often name it em in the examples, but if someone is unfamiliar with jpa, they might have to check what this is - i think the name entityManager is fair enough